docs/checks/approximateWayCheck.md

```diff
@@ -0,0 +1,10 @@
+# ApproximateWayCheck
+
+#### Description
+
+The purpose of this check is to identify ways that are crudely drawn, there is a discrepancy between the drawing and the real way, especially for curves.
+
+#### Code Review
```

Comment:
It would be great if you could add a brief description of some off the more complex logic here. I think an explanation of your findAngle and quadraticBezier methods would be useful.